release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -3441,11 +3441,20 @@ $ oc adm release info 4.15.18 --pullspecs
 The following enhancements are included in this z-stream release:
 
 [id="ocp-4-15-18-unservable-future-versions-status-condition"]
-===== Introducing the UnservableFutureVersions status condition for SHA1 routes
+===== Blocking upgrades for SHA1 serving certificates on routes and Ingress Controllers
 
-* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} causes routes with SHA1 certificates to be rejected.
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to 4.16 cause routes and Ingress Controllers with SHA1 certificates to be rejected.
```

Comment:
Does `{product-version}` get substituted to 4.15? I was thinking that 4.16 doesn't support SHA1 certs, while 4.15 still supports, but sets Upgradeable=False. Does this language matter?